File: GuzzleProject/App/Model/Approval.php
<?php
	
	/**
	 * --------------------------------------------------------------------------
	 *   @Copyright : License MIT 2017
	 *
	 *   @Author : Alexandre Caillot
	 *   @WebSite : https://www.shiros.fr
	 *
	 *   @File : Approval.php
	 *   @Created_at : 10/03/2018
	 *   @Update_at : 10/03/2018
	 * --------------------------------------------------------------------------
	 */
	
	namespace App\Model;
	
	class Approval
	{
		/** @var int */
		protected $id;
		
		/** @var int */
		protected $date;
		
		/** @var Account */
		protected $account;
		
		/** @var Response */
		protected $response;
		
		/**
		 * @return int
		 */
		public function getId(): int
		{
			return $this->id;
		}
		
		/**
		 * @param int $id
		 */
		public function setId(int $id): void
		{
			$this->id = $id;
		}
		
		/**
		 * @return string
		 */
		public function getDate(): string
		{
			$date = new \DateTime();
			$date->setTimestamp($this->date);
			
			return $date->format('d/m/Y');
		}
		
		/**
		 * @param int $date
		 */
		public function setDate(int $date): void
		{
			$this->date = $date;
		}
		
		/**
		 * @return Account
		 */
		public function getAccount(): Account
		{
			return $this->account;
		}
		
		/**
		 * @param Account $account
		 */
		public function setAccount(Account $account): void
		{
			$this->account = $account;
		}
		
		/**
		 * @return Response
		 */
		public function getResponse(): Response
		{
			return $this->response;
		}
		
		/**
		 * @param Response $response
		 */
		public function setResponse(Response $response): void
		{
			$this->response = $response;
		}
	}
?>

File: HerokuProject/src/main/java/shiros/entity/Approval.java
package shiros.entity;

public class Approval {
	protected int id;
	protected Long date;
	protected Account account;
	protected Response response;
	
	public int getId() {
		return id;
	}
	
	public Approval setId(int id) {
		this.id = id;
		return this;
	}
	
	public Long getDate() {
		return date;
	}
	
	public Approval setDate(Long date) {
		this.date = date;
		return this;
	}

	public Account getAccount() {
		return account;
	}
	
	public Approval setAccount(Account account) {
		this.account = account;
		return this;
	}

	public Response getResponse() {
		return response;
	}
	
	public Approval setResponse(Response response) {
		this.response = response;
		return this;
	}
}
